tcollector.py

```diff
@@ -682,6 +688,33 @@ def send_data(self):
         # the packets out of the kernel's queue
 
 
+    def send_data_to_signalfx(self):
+        try:
+            gauges = []
+            for line in self.sendq:
+                split_line = self.add_tags_to_line(line).split()
```

Comment:
Splitting by spaces are safe
metric, value, and timestamp can't have spaces, and tags are checked to not have spaces: https://github.com/optimizely/tcollector/blob/master/tcollector.py#L835